src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -71,6 +71,7 @@ MXNET_OPERATOR_REGISTER_BINARY_WITH_SPARSE_CPU(_backward_sigmoid, kernel_launch_
 MXNET_OPERATOR_REGISTER_UNARY(_copy)
 .MXNET_DESCRIBE("Returns a copy of the input.")
 .add_alias("identity")
+.set_attr<FInferStorageType>("FInferStorageType", ElemwiseStorageType<1, 1>)
```

Comment:
Sorry I just realized that `MXNET_OPERATOR_REGISTER_UNARY ` already set the `FInferStorageType ` attr..

Reply:
So this op should be registered with `MXNET_OPERATOR_REGISTER_UNARY_WITH_SPARSE` instead?